features/add_contact.py

```diff
@@ -15,66 +15,34 @@ def __init__(self, bumblebee_api):
 
     def action(self, spoken_text):
         try:
-            self.add_contact_details()
+            self.add_contact_details2()
```

Comment:
Is this defined?

Reply:
this should have been changed back to `add_contact_details()`